### DISCUSSION TITLE
Problems building for debain trixie / with Qt6Qml module

### DISCUSSION THREAD

**Discussion by josseega** [MiKTeX/miktex#1521] [Q&A]
I am trying to install miktex in a trixie debian, so, building Miktex, nevertheless, there is some problem trying to cmake it, i just cant figure out which module is missing, i have already installed, the modules I think, I would really appreciate some help, this is the problem

`
-- Could NOT find Qt6Qml (missing: Qt6Qml_DIR)
CMake Error at CMakeLists.txt:1125 (find_package):
  Found package configuration file:

    /usr/lib/x86_64-linux-gnu/cmake/Qt6/Qt6Config.cmake

  but it set Qt6_FOUND to FALSE so package "Qt6" is considered to be NOT
  FOUND.  Reason given by package:

  Failed to find required Qt component "Qml".

  Expected Config file at
  "/usr/lib/x86_64-linux-gnu/cmake/Qt6Qml/Qt6QmlConfig.cmake" does NOT exist

  Configuring with --debug-find-pkg=Qt6Qml might reveal details why the
  package was not found.

   find_package search path values and other context for the last package that was not found:
    CMAKE_MODULE_PATH: /home/psichonauta/temp/miktex/cmake/modules;/usr/lib/x86_64-linux-gnu/cmake/Qt6;/usr/lib/x86_64-linux-gnu/cmake/Qt6/3rdparty/extra-cmake-modules/find-modules;/usr/lib/x86_64-linux-gnu/cmake/Qt6/3rdparty/kwin
    CMAKE_PREFIX_PATH:
    $ENV{CMAKE_PREFIX_PATH}:
    CMAKE_FIND_ROOT_PATH:
    _qt_additional_packages_prefix_paths:
    _qt_additional_host_packages_prefix_paths:
    _qt_cmake_dir: /usr/lib/x86_64-linux-gnu/cmake
    QT_HOST_PATH:
    Qt6HostInfo_DIR:
    Qt6_DIR: /usr/lib/x86_64-linux-gnu/cmake/Qt6
    CMAKE_TOOLCHAIN_FILE:
    CMAKE_FIND_ROOT_PATH_MODE_PACKAGE:
    CMAKE_SYSROOT:
    $ENV{PATH}: /home/psichonauta/.cargo/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games
`

**Comment by knight110001** [comment]
sudo apt install qt6-declarative-dev qt6-scxml-dev
